### PR TITLE
[ADD] create new module sale_settings that add a `company_id` field to the sale.config.setting model in order to use related fields in the the configuration wizard

### PR DESCRIPTION
**Pull request by zaoral**
In PR Vauxoo/yoytec#601 is required to count with a `company_id` field in the `sale.config.settings` model. This field exist in `yoytec_supplier_rma_workflow` module.

In order to reuse this field I will move it to a new generic module in the `addons-vauxoo` repository named `sale_settings` and make both `yoytec_sale_workflow` and `yoytec_supplier_rma_workflow` modules depend on it.

Originally, this config wizard has not this field because is implemented in a v7 approach where the company information is extract via context. But taking account module as example we found out that the `account.config.settings` configuration wizard has a `company_id` field that let to manage related fields, and thats, is what we want for now in the `yoytec_sale_workflow` module. 
# TODO
- [x] create new module `sale_settings` basic structure
- [x] copy `company_id` field definition in `yoytec_supplier_rma_workflow` to the `sale_settings` module.
- [x] add unit test to check that the default value of the field is properly set.
- [x] add the module description indicating this new functionality
  sale.config.settings model.
- [x] add module translation.

### CODE REVIEW

**Comment by zaoral** [comment]
@yaniaular @osvalr please review this PR

File: sale_settings/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+===========
+
+Bugs are tracked on
+`GitHub Issues <https://github.com/Vauxoo/addons-vauxoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and
+welcomed feedback
+`here <https://github.com/Vauxoo/addons-vauxoo/issues/new?body=module:%20
+sale_settings%0Aversion:%20
+8.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_
```
Comment (osvalr) [inline sale_settings/README.rst:48]: it is just `8.0`

Comment (zaoral) [inline sale_settings/README.rst:48]: @osvalr please let me know why do you suggest to use only `8.0`?.
I added this version number taking into account the OCA [guideline](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers) version numbers.

**Comment by osvalr** [comment]
Hi @zaoral, I don't see dependency changes made as mentioned in the PR's description for `yoytec_sale_workflow` and `yoytec_supplier_rma_workflow`, but the rest of it looks good to me

**Comment by zaoral** [comment]
@osvalr That is correct. the other two modules that will depend to this one are in another repository so new PRs will be make to link to this added module. This PR in progress.

File: sale_settings/README.rst (hunk 1 above)
Comment (osvalr) [inline sale_settings/README.rst:48]: OCA Guidelines mentions that is for the branch the module belongs.. look over https://github.com/OCA/maintainer-tools/issues/151 for more info

**Comment by yaniaular** [comment]
:+1:

**Comment by zaoral** [comment]
After review this PR with @nhomar we agree that the PR will be dismissed because was not the correct approach.

**Pull request closed without merging**